cli/openshift.go

```diff
@@ -0,0 +1,57 @@
+package cli
+
+import (
+	operatorv1 "github.com/operator-framework/api/pkg/operators/v1"
+	operatorv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/rest"
+)
+
+type OpenShiftOptions struct {
+	Namespace string
+	Labels    map[string]string
+}
+
+type OpenshiftReport struct {
+	Stdout string
+	Stderr string
+	Items  []OperatorSdkScorecardItem `json:"items"`
+}
+
+type SubscriptionData struct {
+	Name                   string
+	Channel                string
+	CatalogSource          string
+	CatalogSourceNamespace string
+	Package                string
+}
+
+type CatalogSourceData struct {
+	Name  string
+	Image string
+}
+
+type OperatorGroupData struct {
+	Name             string
+	TargetNamespaces []string
+}
+
+type OpenshiftEngine interface {
+	CreateNamespace(name string, opts OpenShiftOptions, config *rest.Config) (*OpenshiftReport, error)
```

Comment:
Also, you have inconsistent case. You are using both Openshift, and OpenShift. I think OpenShift is the proper one to use, as that's the product name as well.